js/apps/filter-page/controllers/filterPageCtrl.js

```diff
@@ -135,15 +135,63 @@
 		angular.element(document).on('hide.bs.collapse', '.expando-wrapper .collapse', toggleIcon);
 		angular.element(document).on('show.bs.collapse', '.expando-wrapper .collapse', toggleIcon);
 
-		cardService.get(loadCardsAndFilters);
+		const formatKeyName = (key) => {
+			if (!key) return "";
+
+			return key.replace("-", " ");
+		};
```

Comment:
Hey, you like functional programming, right? ;-)

```
const formatKeyName = (key) => !key ? "" : key.replace("-", " ");
```

Reply:
very nice comment, adjusted